static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -120,6 +124,8 @@ define([
 
         return applyCreativeTemplate(advert.node).then(function (isRendered) {
             return renderAdvertLabel(advert.node)
+                .then(addFeedbackDropdownToggle())
+                .then(applyFeedbackOnClickListeners(slotRenderEvent.creativeId))
```

Comment:
Yes - I missed that one - although in that case you can just get rid of the parentheses: `.then(addFeedbackDropdownToggle)`